platforms/web/lib/constants.ts

```diff
@@ -28,4 +28,6 @@ export const ACTION_TYPES = [
     'link',
     'codeBlock',
     'quote',
+    'indent',
+    'unIndent',
```

Comment:
Does this have to be `unIndent` for any reason? To match something from the rust model? I find it jarring, but if it has to be that way, it has to be that way

Reply:
Yes, it's needed to match the state returned by the rust model